src/main/cms/cms_menu_osd.c

```diff
@@ -205,6 +205,10 @@ static long menuTimersOnEnter(void)
         timerSource[i] = OSD_TIMER_SRC(timer);
         timerPrecision[i] = OSD_TIMER_PRECISION(timer);
         timerAlarm[i] = OSD_TIMER_ALARM(timer);
+        if ((int)timerSource[i] >= (int)OSD_TIMER_SRC_COUNT)
+            timerSource[i] = OSD_TIMER_SRC_ON;
+        if ((int)timerPrecision[i] >= (int)OSD_TIMER_PREC_COUNT)
+            timerPrecision[i] = OSD_TIMER_PREC_SECOND;
```

Comment:
This is not needed. The CMS menu definition constrains the values to `OSD_TIMER_SRC_COUNT - 1` and `OSD_TIMER_PREC_COUNT - 1` respectively.

Reply:
Apparently CMS uses out of range string array indexes...
See screenshot attached to: https://github.com/betaflight/betaflight-configurator/pull/1381